client/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>Morph</artifactId>
```

Comment:
For group and artifact:

* All groupId should be `org.ezalori.morph`;
* Root `pom.xml` artifactId `morph-parent`;
* Sub-module
    * artifactId `morph-web`, `morph-client`;
    * directory `web`, `client`;
    * package `org.ezalori.morph.web.controllers.MyController`, etc.